lib/routes/auth.js

```diff
@@ -1,14 +1,47 @@
 const { Router } = require('express');
+const bcrypt = require('bcryptjs');
+const { check, validationResult } = require('express-validator');
+const User = require('../models/User');
 
 const router = Router();
 
 // /api/auth/register
```

Comment:
This file and router shouldn't know if it is used in `/api/auth` or somewhere else.
I would remove this comment (not related to this PR)

Reply:
Done.